### PR TITLE
PHP 8 compatibility

### PR DESCRIPTION
**Pull request by MidnightDesign**

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
```
Comment (rieschl) [inline phpunit.xml:3]: why that? I'd stick to the relative path as it's always reflecting the currently installed phpunit version (9.5 on `latest`)

**Review by rieschl — COMMENTED** [review]
I'd disable scrutinizer altoghether (removing the .yml won't remove the check)

**Review by rieschl — APPROVED** [review]

File: .github/workflows/checks.yml [hunk 2]
```diff
@@ -41,7 +31,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          composer install --no-progress --no-suggest --no-interaction ${{ env.COMPOSER_ARGS }} > /dev/null
+          composer update --no-progress --no-suggest --no-interaction ${{ matrix.prefer_lowest }} > /dev/null
```
Comment (rieschl) [inline .github/workflows/checks.yml]: `--no-suggest` is deprecated

File: phpunit.xml (hunk 1 above)
Comment (MidnightDesign) [reply phpunit.xml:3]: The path is different in different versions of PHPUnit, so that doesn't work.

File: .github/workflows/checks.yml (hunk 2 above)
Comment (MidnightDesign) [reply .github/workflows/checks.yml]: Thanks! It's gone.

File: phpunit.xml (hunk 1 above)
Comment (rieschl) [reply phpunit.xml:3]: I think it [was a bug](https://github.com/sebastianbergmann/phpunit/issues/4405) (or unintentional BC break), that was fixed in `9.3.4`. So you should be safe to use the relative path for the time being and require phpunit `^9.3.4`

**Pull request merged**